Source/SwiftLintFramework/Extensions/Configuration+Cache.swift

```diff
@@ -71,7 +71,14 @@ extension Configuration {
 
     internal var cacheURL: URL {
         let baseURL: URL
-        if let path = cachePath {
+
+        if ProcessInfo.processInfo.environment["CI"] == "TRUE" {
```

Comment:
CI is a super generic environment variable name. Is there a more reliable way to detect if this is running in an Xcode Cloud environment? I won't want everyone running SwiftLint in their CI to suddenly lose caching.